backend/src/controllers/ServerController.js

```diff
@@ -20,6 +21,9 @@ export default class ServerController {
             const {id} = req.session.user;
```

Comment:
You can easily destructure user email from here. Why getting it from request body?